### PR TITLE
docs: add information about missing opampbridge crd

### PR DESCRIPTION
**Pull request by sumo-drosiek**
### Checklist

- [ ] Changelog updated or skip changelog label added
- [ ] Documentation updated
- [ ] Template tests added for new features
- [ ] Integration tests added or modified for major features

### CODE REVIEW

File: docs/v4-migration-doc.md [hunk 1]
```diff
@@ -136,6 +116,16 @@ kube-prometheus-stack:
...
+#### Apply missing Custom Resource Definition for OpenTelemetry Operator
+
+**When??** If you already uses OpenTelemetry Operator, and it is in version before `0.42.0`
+
+Please apply the following command on your cluster:
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/open-telemetry/opentelemetry-helm-charts/opentelemetry-operator-0.44.0/charts/opentelemetry-operator/crds/crd-opentelemetry.io_opampbridges.yaml
+```
+
```
Comment (swiatekm-sumo) [inline docs/v4-migration-doc.md:128]: If you want to add something here, it should instead go to https://github.com/SumoLogic/sumologic-documentation/blob/main/docs/send-data/kubernetes/v4/how-to-upgrade.md.

Comment (sumo-drosiek) [reply docs/v4-migration-doc.md:128]: We should then link from this document to the Sumo Logic. Now we have duplication which is easy to keep unsynced

Comment (swiatekm-sumo) [reply docs/v4-migration-doc.md:128]: Yes, now done with https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/3477 merged.

Comment (sumo-drosiek) [reply docs/v4-migration-doc.md:128]: https://github.com/SumoLogic/sumologic-documentation/pull/3489/files

Comment (sumo-drosiek) [reply docs/v4-migration-doc.md:128]:
```suggestion
```

**Review by swiatekm-sumo — APPROVED** [review]

**Pull request merged**